packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -276,6 +329,53 @@ const Updates = decorate([
             </CardBlock>
           </Card>
         </Col>
+        <Col mediumSize={6}>
+          <Card>
+            <CardHeader>{_('releaseChannels')}</CardHeader>
+            <CardBlock>
+              <form id='releaseChannelsForm' className='form'>
+                <div className='form-group'>
+                  <Select
+                    autoSelectSingleOption={false}
```

Comment:
Why?

Reply:
`Select` component trigger an `onChange` when the length of the array of the passed options is `1`.
At first, my `Select` receive only `Private channel` option, and then receive the remaining `release channels` from `xoa-updater`. Unfortunetly `Select`  breaks my algorithm by its special onChange trigger.
@pdonias knows more about this.

Reply:
Makes sense, thanks for the explanation.

Maybe we should add a small comment though.